projects/aws/lib/aws-stack.ts

```diff
@@ -152,11 +187,50 @@ export class EditionsStack extends cdk.Stack {
                 ],
             },
         )
+
         new CfnOutput(this, 'Cloudfront-distribution', {
-            description: 'URL for distribution',
+            description: 'URL for backend',
             value: `https://${dist.domainName}`,
         })
 
+        const previewGateway = new apigateway.LambdaRestApi(
+            this,
+            'editions-preview-backend-apigateway',
+            {
+                handler: previewBackend,
+            },
+        )
+
+        const previewGatewayId = previewGateway.restApiId
+
+        const previewDist = new cloudfront.CloudFrontWebDistribution(
```

Comment:
Can we have a builder for these if they are very similar `preview` and `prod`? (Or remove as discussed).